sympy/physics/optics/medium.py

```diff
@@ -152,6 +131,62 @@ def refractive_index(self):
         """
         return (c/self.speed)
 
+    def __lt__(self, other):
+        """
+        Compares based on refractive index of the medium.
+        """
+        return self.refractive_index < other.refractive_index
+
+    def __gt__(self, other):
+        return not self < other
+
+    def __ne__(self, other):
+        return not self == other
+
+
+class MediumN(Medium):
+
+    def __new__(cls, name, n):
```

Comment:
At least a short class docstring could explain what this class is and show two to create it (by calling `Medium`)

Reply:
Same goes for `MediumPP`.